hastory-server-data/src/Hastory/Server/Data/UserForm.hs

```diff
@@ -28,6 +27,3 @@ instance ToJSON UserForm where
 
 instance Validity UserForm where
   validate = delve "userFormUserName" . userFormUserName
```

Comment:
The validity instance can just be derived now. Just remove the implementation.

Reply:
Removing the implementation leads to a `No instance for (GenUnchecked UserForm)`. I'm still not done with this part of the code so I will be revisiting.

Reply:
As discussed earlier, we can't use the the generic derived `Validity UserForm` because there is not instance of `GenUnchecked UserForm`.